---
# bool success
# geometry_msgs/Pose pose
string res_to_print